int16 green_value
int16 blue_value
int16 yellow_value
int16 red_value
